osgar/drivers/tankmaria.py

```diff
@@ -0,0 +1,167 @@
+"""
+  Driver for a small tank Maria
+
+  Text Communication Protocol
+  Version 0 uses ASCII protocol between the host computer (JetsonNano) and Arduino motor controller.
+  There is no speed control implemented on Arduino at the moment, so raw PWM is used:
+     "<PWM left> <PWM right> <desired LED status>\n"
+  The value for PWM can in in -255..255.
+  LED status: 0=OFF, 1=green, 2=red.
+
+  The board regularly responds (20Hz?) with status line:
+      "<32bit unsigned int timestamp in milliseconds> <encoder left> <encoder right> <buttons mask>\n"
+  The encoders values are 32bit signed int and accumulative.
```

Comment:
Just a nitpick - can we have just names in the string definition and all types outside? Ergo change `timestamp` to be like encoders.